docs/content/en/docs/reference/helm-chart.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Helm chart"
```

Comment:
Please make this: Kubernetes Helm chart  , so it is clear for everyone.

Reply:
Thanks for the review, however, I don't agree, what is a non-Kubernetes helm chart? I try to keep it short for the summary navbar on the left and don't want to introduce unnecessary stuff.